src/kata7.getEmployerRole.js

```diff
@@ -1,5 +1,9 @@
 const getEmployerRole = (employeeName, employees) => {
```

Comment:
You're very close in nailing the implementation of this one: 
- first error is `employee is not defined` and that's because it needs to be declared like ```for (let employee of employees)``` 💡 notice the `let`
- then our comparison in the `if` statement: `employee` is the whole object that includes `name` and `role`, so we might want to write it like `if (employee.name === employeeName)` 
- finally, we want to return the employee role, and we can do that by `return employee.role`